examples/manually_deploy_all_applications.sh

```diff
@@ -47,14 +54,18 @@ manual_deploy_node_app() {
 # Variable definition
 #####################
 check_variable_is_set CF_SPACE
-check_variable_is_set BASE_DIR "The directory containing the continuous delivery project, e.g. ~/htbhf-continous-delivery"
+check_variable_is_set BASE_DIR "The parent directory of the continuous delivery project"
 export CD_SCRIPTS_DIR="${BASE_DIR}/htbhf-continous-delivery/cd_scripts"
 export WORKING_DIR="${BASE_DIR}/temp-deploy"
 export BIN_DIR="${WORKING_DIR}/bin"
 export BINTRAY_ROOT_URL=https://dl.bintray.com/departmentofhealth-htbhf/maven/uk/gov/dhsc/htbhf
 
 
 # get versions of code to deploy from: https://departmentofhealth-htbhf.github.io/htbhf-continous-delivery/docs/production_app_versions.txt
+wget -q https://departmentofhealth-htbhf.github.io/htbhf-continous-delivery/docs/production_app_versions.txt
+cat production_app_versions.txt | tr '-' '_' | tr '[:lower:]' '[:upper:]' | sed -e 's/\([A-Z0-9_]*\):\s*/export \1_VERSION=/g' > set_app_versions.sh
+source ./set_app_versions.sh
+
```

Comment:
Downloads the deployed version numbers and converts then into environment variables